optional-requirements.txt

```diff
@@ -1 +1 @@
-autoit==0.2
+clipboard==0.0.4
```

Comment:
I took a look at the source code. It seems to be a tiny shim around `pyperclip`. Can we use that directly instead?

Reply:
At a glance, `pyperclip` looks safe too. It does have some comments regarding executing programs by name on Linux, but if that's compromised, there's probably bigger issues. It's good that they document this too.

Reply:
pyperclip and clipboard both work, I've used both. Clipboard is just my goto, if only because the name is easier to remember. I don't have any preference, so I'll just switch it to pyperclip real fast.